tests/fixtures/project_with_tasks_that_handle_interrupts/loop_with_sigint_handler.py

```diff
@@ -1,11 +1,21 @@
+import signal
```

Comment:
Don't be shy, create another fixture for your test ;) You can copy this directory and rename it, but I would much rather that we have an explicit fixture for the sigint test, and another for the general signals handoff.

Why is that? because in the SIGINT case, the signal is actually sent to the inner process directly, and in our case we send the signal to the parent process and expect it to drill down. Two different cases, two different tests and therefore two different fixtures :)